Pipeline/CreateUCDComponentVersion/README.md

```diff
@@ -14,15 +14,17 @@ This section provides a more detailed explanation of how the CreateUCDComponentV
 
 1. **Initialization**
    1. Read command line parameters.
-   1. Read any application and global properties that are passed as a comma-separated list via `--packagingPropFiles` (This is optional for the UCD package format v1, but an required option using the UCD package format v2 - see `--ucdV2PackageFormat`).
+   1. Read any application and global properties files that are passed as a comma-separated list via `--packagingPropFiles`.
```

Comment:
Added "files" here, as I believe `--packagingPropFiles` takes files rather than the individual properties themselves. (Please feel free to correct this if it is wrong.)